inc/Engine/CDN/RocketCDN/views/cta-big.php

```diff
@@ -63,33 +78,17 @@
 					<h4 class="wpr-title2 wpr-rocketcdn-pricing-regular"><del>$<?php echo esc_html( $data['regular_price'] ); ?></del></h4>
 					<?php endif; ?>
 					<h4 class="wpr-rocketcdn-pricing-current">
-						<?php
-						printf(
-							// translators: %s = price of RocketCDN subscription.
-							esc_html__( '%s / month', 'rocket' ),
-							'<span class="wpr-title1">$' . esc_html( $data['current_price'] ) . '</span>'
-						);
-						?>
+						<span class="wpr-rocketcdn-cta-currency-minor">$</span>
+						<span class="wpr-rocketcdn-cta-currency-major"><?php esc_html_e( substr( $data['current_price'], 0, strpos( $data['current_price'], '.' ) ) ); ?></span>
+						<span class="wpr-rocketcdn-cta-currency-minor"><?php esc_html_e( substr( $data['current_price'], strpos( $data['current_price'], '.' ) ) ); ?>
+						<?php if ( ! empty( $data['regular_price'] ) ): ?>*<?php endif; ?>
```

Comment:
What is the use of this line, this is adding two `*` when discount is there, or what am I missing here?
![image](https://user-images.githubusercontent.com/15707971/145195243-12a5a8e5-684d-4411-ac6d-7ce16830c6f3.png)